demos/palm/python/docs-agent/apps_script/README.md

```diff
@@ -0,0 +1,164 @@
+# Convert Google Docs, PDF, and Gmail to Markdown files
+
+The collection of scripts in this `apps_script` directory allows you to convert
+the contents of Google Drive folders and Gmail to Markdown files that are
+compatible with Docs Agent.
+
+The steps are:
+
+1. [Prepare a Google Drive folder](#1-prepare-a-google-driver-folder).
+2. [Mount Google Drive on your host machine](#2-mount-google-drive-on-your-host-machine).
+3. [Create an Apps Script project](#3-create-an-apps-script-project).
+4. [Edit and run main.gs on Apps Script](#4-edit-and-run-main-gs-on-apps-script).
+5. [Update config.yaml to include the mounted directory](#5-update-config-yaml-to-include-the-mounted-directory).
+
+## 1. Prepare a Google Drive folder
+
+First, create a new folder in Google Drive and add your Google Docs (which will be
+used as source documents to Docs Agent) to the folder.
+
+Do the following:
+
+1. Browser to https://drive.google.com/.
+1. Click **+ New** on the top left corner.
+1. Click **New folder**.
+1. Name your new folder (for example, `my source Google Docs`).
+1. To enter the newly created folder, double click the folder.
+1. Add (or move) your source Google Docs to this new folder.
+
+## 2. Mount Google Drive on your host machine
+
+Mount your Google Drive to your host machine, so that it becomes easy to access the
+folders in Google Drive from your host machine (later in step 5).
+
+There are a variety of methods and tools available online that enable this setup
+(for example, see [`google-drive-ocamlfuse`][google-drive-ocamlfuse] for Linux machines).
+
+## 3. Create an Apps Script project
+
+Create a new Apps Script project and copy all the `.gs` scripts in this
+`apps_script` directory to your new Apps Script project.
+
+Do the following:
+
+1. Browse to https://script.google.com/.
+1. Click **New Project**.
+1. At the top of the page, click **Untitled Project** and enter a meaningful
+   title (for example, `gDocs to Docs Agent`).
+1. Click the **+** icon next to **Files**.
+1. Click **Script**.
+1. Name the new script to be one of the `.gs` files in this `apps_script` directory
+   (for example, `drive_to_markdown`).
+1. Copy the content of the `.gs` file to the new script on your Apps Script project.
+1. To save, click the "Save project" icon in the toolbar.
+1. Repeat the steps until all the `.gs` files are copied to your Apps Script project.
+1. Click the **+** icon next to **Services**.
+1. Scroll down and click **Drive API**.
+1. Click **Add**.
+
+You are now ready to edit the parameters on the `main.gs` file to select a folder
+in Google Drive and export emails from Gmail.
+
+![Apps Script project](../docs/images/apps-script-screenshot-01.png)
```

Comment:
This PNG image seems to be missing. But it's not a blocker for this pull request -- we can consider adding the image in the next pull request.